### ISSUE TITLE
Consider removing `unpack_sequence` from our list of benchmarks

### ISSUE THREAD

**Issue by Fidget-Spinner** [faster-cpython/benchmarking-public#35]
It's a microbenchmark and skews numbers unnecessarily. 

Here's the code https://github.com/python/pyperformance/blob/main/pyperformance/data-files/benchmarks/bm_unpack_sequence/run_benchmark.py

Here's an example benchmark run https://github.com/faster-cpython/benchmarking-public/blob/main/results/bm-20240330-3.13.0a5%2B-790b1f8-JIT/bm-20240330-pythonperf2-x86_64-brandtbucher-justin_align-3.13.0a5%2B-790b1f8-vs-base.md . Unpack_sequence is the clear outlier here.

**Comment by gvanrossum** [comment]
Agreed, we've been complaining about this benchmark long enough.

**Comment by mdboom** [comment]
Yes, I agree.  If we want to do this, we should probably file a PR against `pyperformance` and get at least one non-Faster CPython reviewer to approve it there.
